index.js

```diff
@@ -383,31 +383,53 @@ module.exports = {
   _getDebugMacroPlugins(config) {
     let addonOptions = config['ember-cli-babel'] || {};
 
-    if (addonOptions.disableDebugTooling) { return; }
+    if (addonOptions.disableDebugTooling) {
+      return;
+    }
 
     const isProduction = process.env.EMBER_ENV === 'production';
     const isDebug = !isProduction;
 
-    let options = {
-      flags: [
+    return [
+      [
+        require.resolve('babel-plugin-debug-macros'),
         {
-          source: '@glimmer/env',
-          flags: { DEBUG: isDebug, CI: !!process.env.CI }
-        }
+          flags: [
+            {
+              source: '@glimmer/env',
+              flags: { DEBUG: isDebug, CI: !!process.env.CI },
+            },
+          ],
+
+          externalizeHelpers: {
+            global: 'Ember',
+          },
+
+          debugTools: {
+            isDebug,
+            source: '@ember/debug',
+            assertPredicateIndex: 1,
+          },
+        },
+        '@ember/debug stripping',
       ],
-
-      externalizeHelpers: {
-        global: 'Ember'
-      },
-
-      debugTools: {
-        isDebug,
-        source: '@ember/debug',
-        assertPredicateIndex: 1
-      }
-    };
-
-    return [[require.resolve('babel-plugin-debug-macros'), options]];
+      [
+        require.resolve('babel-plugin-debug-macros'),
```

Comment:
running the same plugin twice seems unfortunate, can the primary plugin simply be taught how to handle this case?

Reply:
We already support arbitrary module paths with `flags`, but not with `debugTools`. We can definitely update `babel-plugin-debug-macros` to support that though.

Reply:
i think this may be tripping up https://github.com/babel/babel/blob/11fa2461cec930168fd43cfeda733aab056d8318/packages/babel-core/src/config/config-descriptors.js#L240

Reply:
investigating

Reply:
The issue seems to have been a monorepo with a shared lockfile resulted in an incomplete semver drift of  relevant dependencies, upgrading all repos contained within the monorepo fixed the issues.